.github/pull_request_template.md

```diff
@@ -1,36 +1,32 @@
-# Motivation
+# Description[^1]
 
-Short explanation on why this change is needed and reference to ticket (link created automatically): TICKET-123
```

Comment:
I really liked TICKET-123

Reply:
I am with @sanderkoiv to have reference to ticket